third_party/accessibility/BUILD.gn

```diff
@@ -75,6 +75,20 @@ if (enable_unittests) {
         "ax/platform/test_ax_node_wrapper.h",
       ]
 
+      if (is_mac) {
+        sources += [ "ax/platform/ax_platform_node_mac_unittest.mm" ]
+        frameworks = [
+          "AppKit.framework",
+          "CoreFoundation.framework",
+          "CoreGraphics.framework",
+          "CoreText.framework",
+          "IOSurface.framework",
+        ]
+
+        #cflags_objcc = flutter_cflags_objcc_arc
```

Comment:
Should this be removed?

Reply:
Uncommented, in fact! (And deleted the following line)